Makefile

```diff
@@ -367,8 +367,25 @@ upload-db:
 TARGET ?= staging
 
 
-release-list release deploy:
-	@echo $@ not yet implemented
+release-list:
+	@git ls-remote --tags | grep -o 'refs/tags/v[0-9]*\.[0-9]*\.[0-9]*' | sort -r | grep -o '[^\/]*$$'
+
+
+install-releasedeps:
+	@if [ -d "$DIRECTORY" ]; then \
+		source .venv/bin/activate; \
+	else \
+		virtualenv --no-site-packages .venv && source .venv/bin/activate; \
+	fi;
+	@pip install python-semantic-release
```

Comment:
I really don't like the fact that this requires working with virtualenv like this. Can't this happen inside a container?

At the very list, it means that the pre-requisites for this project should include installing virtualenv.

Reply:
fixed - wrapped the semantic-release module into a dockerfile. you will need to re-run ```make build``` to create the image before running ```make release```